Dockerfile

```diff
@@ -41,7 +41,7 @@ FROM builder-py-base AS builder-py-3_13t
 RUN /build_python.sh 3.13.2t
 # ------------------------------------------------------------------------------
 FROM builder-py-base AS builder-py-3_14
-RUN /build_python.sh 3.14-dev
+RUN /build_python.sh 3.14.0b2
```

Comment:
patch ver & extras are ignored by the install script, so this doesn't change the file path that it is installed to, which is `/snekbin/python/3.14/`